lasagne/init.py

```diff
@@ -25,6 +65,22 @@ def sample(self, shape):
 
 
 class Uniform(Initializer):
+    """Sample initial weights from the uniform distribution
+
+    Parameters are sampled from U(a, b).
+
+    Parameters
+    ----------
+    range : float or tuple
+        When std is None then range determines a, b. If range is a float the
+        weights are sampled from U(-range, range). If range is a tuple the
+        weights are sampled from U(range[0], range[1]).
+    std : float or None
+        If std is a float then the weights are sampled from
+        U(mean - np.sqrt(3) * std, mean + np.sqrt(3) * std).
+    mean : float
+        see std for description.
+    """
```

Comment:
I think it's a good second use of the `mean` parameter now that we have it, it makes the documentation a bit easier, and it won't affect existing code. But I can add this after this PR.